regression/validate-trace-xml-schema/validate-xsd/src/main/java/com/diffblue/validatexsd/LoggingErrorHandler.java

```diff
@@ -0,0 +1,30 @@
+package com.diffblue.validatexsd;
+
+import org.xml.sax.ErrorHandler;
+import org.xml.sax.SAXException;
+import org.xml.sax.SAXParseException;
+
+public class LoggingErrorHandler implements ErrorHandler {
+  private boolean errorOccurred = false;
+
+  @Override
+  public void warning(SAXParseException e) throws SAXException {
```

Comment:
Should you be re-throwing these execptions? Seems weird the signature would throw otherwise?

Reply:
You can rethrow, but you don't have to.